ceph.spec.in

```diff
@@ -14,7 +14,7 @@
 #
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 # 
-%bcond_with ocf
+%bcond_without ocf
```

Comment:
maybe the package name of "resource-agents" is too general, and there is an existing package named exactly the same, which is "Required" by our "resource-agents" as put by the "ceph.spec.in". see https://github.com/ClusterLabs/resource-agents/blob/master/resource-agents.spec.in and http://centos-packages.com/7/package/resource-agents/ . shall we rename it to ceph-resource-agent?

seems opensuse has renamed it already? https://rpmfind.net/linux/RPM/opensuse/updates/leap/42.2/oss/x86_64/ceph-resource-agents-10.2.4+git.1481215985.12b091b-4.1.x86_64.html @smithfarm is that you ? =)